package.json

```diff
@@ -4,43 +4,44 @@
   "description": "A polyfill for the proposed blocking elments stack API",
   "main": "dist/blocking-elements.js",
   "author": "Valdrin Koshi <valdrin@google.com>",
-  "license": "BSD-3-Clause",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/PolymerLabs/blockingElements.git"
+    "url": "git+https://github.com/PolymerLabs/blocking-elements.git"
   },
   "keywords": [
     "blocking-elements",
     "polyfill",
     "browser"
   ],
   "bugs": {
-    "url": "https://github.com/PolymerLabs/blockingElements/issues"
+    "url": "https://github.com/PolymerLabs/blocking-elements/issues"
   },
-  "homepage": "https://github.com/PolymerLabs/blockingElements#readme",
+  "homepage": "https://github.com/PolymerLabs/blocking-elements#readme",
   "scripts": {
-    "lint": "eslint blocking-elements.js",
+    "lint": "tslint --project ./",
     "test": "npm run lint && npm run build && easy-sauce",
-    "build": "rollem"
+    "format": "clang-format --style=file -i src/*.ts",
+    "prepack": "npm run build",
+    "build": "rm dist/* -f && tsc && rollup -c && rm dist/document-global.*"
```

Comment:
Actually I wanted to `rm dist/*` instead of `rm dist` on on purpose because in the past I've been bitten by tools that are watching the directory's *inode* getting confused by the directory going away. Rewrote a little though.